website/.vscode/extensions.json

```diff
@@ -0,0 +1,8 @@
+{
+    "recommendations": [
+        "dbaeumer.vscode-eslint",
+        "unifiedjs.vscode-mdx",
+        "esbenp.prettier-vscode",
+        "syler.sass-indented"
+    ]
+}
```

Comment:
Sorry if I missed earlier conversations on this, can you provide a bit more background information about these?

Reply:
Thanks!

Reply:
Also VSCode will offer you to automatically activate these extensions if you choose so. Especially when switching projects a lot this is quite handy.